docs/BUILD.md

```diff
@@ -0,0 +1,253 @@
+   if (keystorePropertiesFile.exists()) {
+       keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+   }
+
+   android {
+         ...
+   }
+```
+
+Load the `key.properties` file into the `keystoreProperties` object.
+
+2. Add code before buildTypes block:
+
+```
+   buildTypes {
+       release {
+           // TODO: Add your own signing config for the release build.
+           // Signing with the debug keys for now,
+           // so `flutter run --release` works.
+           signingConfig signingConfigs.debug
+       }
+   }
+```
+
+With the signing configuration info:
+
+```
+   signingConfigs {
+       release {
+           keyAlias keystoreProperties['keyAlias']
+           keyPassword keystoreProperties['keyPassword']
+           storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
+           storePassword keystoreProperties['storePassword']
+       }
+   }
+   buildTypes {
+       release {
+           signingConfig signingConfigs.release
+       }
+   }
+```
+
+Configure the `signingConfigs` block in your module’s `build.gradle` file.
+
+Release builds of your app will now be signed automatically.
+
+### Reviewing the app manifest
+
+Review the default [App Manifest](https://developer.android.com/guide/topics/manifest/manifest-intro) file, `AndroidManifest.xml`, located in `client/flutter/android/app/src/main` and verify that the values are correct, especially the following:
+
+`application`
+
+Edit the `android:label` in the application tag to reflect the final name of the app.
+
+### Reviewing the build configuration
+
+Review the default [Gradle build file](https://developer.android.com/studio/build/#module-level) file, `build.gradle`, located in `client/flutter/android/app` and verify the values are correct, especially the following values in the `defaultConfig` block:
+
+`applicationId`
+
+Specify the final, unique (Application Id)appid
+
+`versionCode` & `versionName`
+
+Specify the internal app version number, and the version number display string. You can do this by setting the version property in the pubspec.yaml file. Consult the version information guidance in the [versions documentation](https://developer.android.com/studio/publish/versioning).
+
+### Building the app for release
+
+These instructions use the App Bundle release format instead of the APK format.
+
+#### Build an app bundle
+
+`cd` into the `client/flutter` directory:
+
+```
+cd .. # this assumes you're in the client/flutter/ios directory, if you're in the app directory you'll want to cd client/flutter
+```
+
+Build the app bundle with flutter:
+
+```
+flutter build appbundle
+```
+
+The release bundle for your app is created at `client/flutter/build/app/outputs/bundle/release/app.aab`.
+
+### Upload the bundle to Google Play
+
+This adapts instructions found [here](https://support.google.com/googleplay/android-developer/answer/7159011).
```

Comment:
Replace lines 214 to 247 with just the link to the Google Play instructions. This follows the important DRY principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

With a copy, the problem is when the instructions change. Google's documentation is automatically updated but the documentation here is not. So the goal is to have the WHO app specific comments here and then refer to the other documentation elsewhere

Reply:
Most important difference to normal releases is:
> 4. Next to the release type you want to create, select **Manage**, then select **Closed**.

to keep it from going live